core/transformer/src/IModelImporter.ts

```diff
@@ -487,6 +497,18 @@ export class IModelImporter implements Required<IModelImportOptions> {
       }
     }
   }
+
+  /** Examine the geometry streams of every [GeometricElement3d]($backend) in the target iModel and apply the specified optimizations.
+   * @note This method is automatically called from [[IModelTransformer.processChanges]] and [[IModelTransformer.processAll]] if
+   * [[IModelTransformOptions.optimizeGeometry]] is defined.
+   */
+  public optimizeGeometry(options: OptimizeGeometryOptions): void {
+    if (options.inlineUniqueGeometryParts) {
+      const result = this.targetDb.nativeDb.inlineGeometryPartReferences();
+      if (result.numCandidateParts > 0)
+        Logger.logInfo(loggerCategory, `Inlined ${result.numRefsInlined} references to ${result.numCandidateParts} geometry parts and deleted ${result.numPartsDeleted} parts.`);
```

Comment:
I guess we generally don't log if no action is taken, but I think I'd prefer having a breadcrumb log that this process found no candidate parts.

This is a nit, though - feel free to resolve my comment if you disagree.

Reply:
You're right - if someone requested optimization we should log the results unconditionally. Made that change.